homeassistant/components/onewire/sensor.py

```diff
@@ -246,9 +277,13 @@ def get_entities(onewirehub: OneWireHub, config):
             device_type = device["type"]
             device_id = os.path.split(os.path.split(device["path"])[0])[1]
             dev_type = "std"
```

Comment:
Can you please take the opportunity to rename the variable?
```suggestion
            device_sub_type = "std"
```